Sources/CoverageHighlighter/CoverageHighlighter.swift

```diff
@@ -108,26 +121,62 @@ public class CoverageHighlighter {
         let executionCount: Int
     }
 
-    public static func hightlight() -> Void {
+    public static func hightlight(filter: Filter) -> Void {
+
         let danger = Danger()
         let filename = "results.json"
         let parser = Parser()
 
+        let include = filter.include
+        let exclude = filter.exclude
         let result = parser.parse(filename, shouldPrint: false)
-        let toBeTested = result?.targets.map { $0.files }.flatMap { $0 }.filter { ($0.name.contains("Service") || $0.name.contains("Presenter") || $0.name.contains("Manager")) && !$0.name.contains("Test")  }.sorted(by: { $0.lineCoverage > $1.lineCoverage }).map { $0 } ?? [File]()
-        let modifiedFilesNames = (danger.git.modifiedFiles + danger.git.createdFiles).compactMap { $0.components(separatedBy: "/").last }
 
-        print("Coverage in on the modified Servicies and Presenters")
+        let filesToHighlightCoverage = result?.targets.map { $0.files }.flatMap { $0 }.filter { codeFile in
+            include.contains(where: { codeFile.name.contains($0) }) &&
+            !exclude.contains(where: { codeFile.name.contains($0) })
+        }.sorted(by: { $0.lineCoverage > $1.lineCoverage }).map { $0 } ?? [File]()
+
+        let modifiedFilesNames = (danger.git.modifiedFiles + danger.git.createdFiles).compactMap { $0.components(separatedBy: "/").last }
 
-        toBeTested.filter { file in
+        let modifiedFilesToHightLight = filesToHighlightCoverage.filter { file in
             modifiedFilesNames.contains(file.path.components(separatedBy: "/").last!)
-        }.forEach {
-            let finalMessage = String(format: "%@: %.3f%%", $0.path.components(separatedBy: "/").last!.padding(toLength: 50, withPad: " ", startingAt: 0), $0.lineCoverage * 100)
-            if $0.lineCoverage * 100 < 70 {
+        }
+
+        modifiedFilesToHightLight.forEach { file in
+
+            typealias ResultFilter = (functions: [Filter.FilterFile], files: [Filter.FilterFile])
+
+            let result: ResultFilter  = ([],[])
+
+            let functions: ResultFilter = filter.filesToFilter.reduce(result) { (result, filterFile) -> ResultFilter in
```

Comment:
This function basically prints the coverage for the functions inside the files we want to hightlight.